catalog/spatial/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -29,12 +29,12 @@ The `HTTP GET` form of `GetCapabilities` uses query parameters via the following
 
```

Comment:
All the other GetCapabilities in this section were flagged `GetCapabilities` except GetCapabilities `HTTP GET`